third_party/address_sorting/address_sorting.c

```diff
@@ -0,0 +1,369 @@
+/*	$NetBSD: getaddrinfo.c,v 1.82 2006/03/25 12:09:40 rpaulo Exp $	*/
```

Comment:
The PR seems to indicated that address_sorting will be a submodule (e.g. linking to some other git repo), which it is not - the code in imported in our repository actually.
That actually matters for cmake, because depending if this is an actual third_party dependency  or just imported code changes the way thing should be done in CMakeLists.txt

if this is an actual "separate library", then what I suggested in the previous comment holds.
If not (which seems to be the case), then you can just define a cmake target address_sorting (and you're doing that) and if that target is properly configured, you don't need to deal with the include directories at all - cmake will do that automatically for you.
Also, the dependency on address_sorting (defined in build.yaml) shouldn't require extra changes in the CMakeLists.txt - the right address_sorting target should be generated automatically and it should be added as a dependency where needed (based on the info from build.yaml file).

Reply:
Addresses in above comment. I took the `cmake/address_sorting.cmake` reogranization suggestion, but I believe I can't use only `build.yaml` for this per description in my above comment.